pep-9999.rst

```diff
@@ -261,15 +261,19 @@ type is assumed to be ``Any``. It is preferred to leave unknown
 types unannotated rather than explicitly marking them as ``Any``, as some
 type checkers can optionally warn about unannotated arguments.
 
-The type of
-an argument where the annotation and the type of the default argument disagree
-is unspecified. As an exception, the ellipsis literal can stand in for any
-type::
-
-    # x has type Any
-    # y's type is unspecified
-    # z has type int
-    def foo(x, y: int = "", z: int = ...): ...
+If an argument has a default value, it must match the implementation
+and the type of the argument (if specified) must match the default value.
+Alternatively, ``...`` can be used in place of any default value::
+
+    # a has type Any
+    # b's default argument is invalid
+    # c and d have type int
```

Comment:
I'm not sure what these first three comments refer to?

Reply:
Oops, these were comments from the previous version that I forgot to delete.